tools/vf-component-library/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 * Bug fix: Fixes identical changelog information for all components changelog section #1874
 * Feature: Eleventy upgrade to version 2.0.0
+* Added: Angular experimental support for vf-badge component
+* Added: Angular experimental support for vf-banner component
+* Added: Angular experimental support for vf-hero component
```

Comment:
Why is this component-library changelog updated with angular changes ?

Reply:
In the PR https://github.com/visual-framework/vf-core/pull/1906/files I have changes in 
- tools/vf-component-library/src/site/_includes/layouts/components.njk 
So  that's why added CHANGELOG.md in this PR.

I think I'll also need to update the PR with tools/vf-core/CHANGELOG.md for the below change (from same PR)
- tools/vf-core/fractal.js